superset/reports/schemas.py

```diff
@@ -160,7 +161,10 @@ class ReportSchedulePostSchema(Schema):
     validator_config_json = fields.Nested(ValidatorConfigJSONSchema)
     log_retention = fields.Integer(description=log_retention_description, example=90)
```

Comment:
Can we add the positive integer validation for `log_retention` and `working_timeout` on this PR?